pyrtl/helperfuncs.py

```diff
@@ -150,7 +127,23 @@ def probe(w, name=None):
     confuse various post-processing transforms such as output to verilog)
     """
     global probeIndexer
-    return probeIndexer.next_indexed(w, name)
+    index = probeIndexer.next_index()
+    prefix = probeIndexer.internal_prefix
+
+    if not isinstance(w, WireVector):
+        raise PyrtlError('Only WireVectors can be probed')
+
+    print('(%s-%d)' % (prefix, index), end=' ')
+    print(get_stack(w))
+
+    if name:
+        pname = name
+    else:
+        pname = '(%s%d__%s)' % (prefix, index, w.name)
```

Comment:
think of a way to simplify the previous 4 lines (changing a variable name is fine)